### PR TITLE
Add captions for ElfProgramHeader

### PR DESCRIPTION
**Pull request by whyitfor**
**One sentence summary of this PR (This should go in the CHANGELOG!)**
Add a caption to ElfProgramHeader.

**Please describe the changes in your request.**
This is a small quality-of-life improvement for the GUI:

image

**Anyone you think should look at this, specifically?**

### CODE REVIEW

File: ofrak_core/ofrak/core/elf/model.py [hunk 1]
```diff
@@ -240,6 +242,9 @@ class ElfProgramHeaderType(Enum):
     SHLIB = 5
     PHDR = 6
     TLS = 7
+    GNU_EH_FRAME = 0x6474E550
+    GNU_STACK = 0x6474E551
+    GNU_RELRO = 0x6474E552
```
Comment (rbs-jacob) [inline ofrak_core/ofrak/core/elf/model.py:247]: Where do these numbers come from? A comment with a citation would be useful.

**Review by rbs-jacob — CHANGES_REQUESTED** [review]
Looks good, with the exception of one small comment!

File: ofrak_core/ofrak/core/elf/model.py (hunk 1 above)
Comment (rbs-jacob) [reply ofrak_core/ofrak/core/elf/model.py:247]: Perhaps something like this?

<https://github.com/torvalds/linux/blob/ceaa837f96adb69c0df0397937cd74991d5d821a/include/uapi/linux/elf.h#L25-L41>

Comment (rbs-jacob) [reply ofrak_core/ofrak/core/elf/model.py:247]:
```suggestion
    # Magic numbers from:
    # https://elixir.bootlin.com/linux/latest/source/include/uapi/linux/elf.h#L38
    GNU_EH_FRAME = 0x6474E550
    GNU_STACK = 0x6474E551
    GNU_RELRO = 0x6474E552
```

Comment (whyitfor) [reply ofrak_core/ofrak/core/elf/model.py:250]: Added a link in the docstring

**Pull request merged**